pages/favs.jsx

```diff
@@ -3,18 +3,64 @@ import { getFavs } from '../js/favs-manager';
 import ProjectList from '../components/project-list/project-list.jsx';
 import HintMessage from '../components/hint-message/hint-message.jsx';
 
+import Service from '../js/service.js';
+import config from '../config/config.js';
+import env from "../config/env.generated.json";
+
 export default React.createClass({
   getInitialState() {
     return {
-      favs: null
+      favs: null,
+      loggedInCheckDateTime: null
     };
   },
+  verifyLoggedInStatus() {
+    Service.nonce()
```

Comment:
There is a new decidated `/userstatus/` route now, so we should probably switch this from `.nonce` to `.userStatus`